web/src/components/buttonBlue/styles.scss

```diff
@@ -0,0 +1,17 @@
+.button {
+  background-color: #546de5;
+  color: #fff;
+  font-size: 16px;
+  font-weight: bold;
+  padding: 5px 25px;
+  border-radius: 5px;
+  margin: 10px 0px;
```

Comment:
precisa dessa margem ? porque se for utilizar esse componentem em outro lugar ele pode ficar estranho

Reply:
Retirei a margin. 
A borda ta 5px, Qual a correta?
![image](https://user-images.githubusercontent.com/62192072/132045887-144b9290-902f-4b19-8abb-ee4a41e23867.png)